release-notes/release-notes58.md

```diff
@@ -101,6 +101,29 @@ The {{site.productname}} 5.8 release includes an accompanying release of the **E
 
 For information on the Export plugin, see: [Export plugin]({{site.baseurl}}/plugins/premium/export/).
 
+### PowerPaste 5.5.0
+
+The {{site.productname}} 5.8 release includes an accompanying release of the **PowerPaste** premium plugin.
+
+This release adds support for copying and pasting content from Google Docs, making it possible to copy and retain styles from Google Docs content. By default, a dialog will now appear asking users if the content should be cleaned or merged when pasting Google Docs content.
+
+**PowerPaste** 5.5.0 provides the following new features:
+
+- Added Google Docs paste support which can be controlled using the new [`powerpaste_googledocs_import`]({{site.baseurl}}/plugins/premium/powerpaste/#powerpaste_googledocs_import) option.
+
+**PowerPaste** 5.5.0 provides the following improvements:
+
+- The editor will now show a "loading" screen while paste events are processed.
+
+**PowerPaste** 5.5.0 provides the following bug fixes:
+
+- Fixed an issue where automatic linking didn't work with URLs containing commas in the path.
```

Comment:
```suggestion
- Fixed an issue where automatic linking didn't work for URLs containing commas.
```

Reply:
This is the same as what's distributed in the actual changelog. Are you sure you want it changed?

Reply:
dang. ok ignore me